### PR TITLE
Updated for best practices & coding standards.

### PR DESCRIPTION
**Pull request by jrfnl**
- Added file header
- Localized and escaped the text string
- Applied best practices and coding standards
- Added readme.txt
- Added license file

Tested & found working in WP 4.5

These changes are the same as the ones contained in https://github.com/TGMPA/TGM-Plugin-Activation/pull/557
Once this PR is merged, PR https://github.com/TGMPA/TGM-Plugin-Activation/pull/557 can be merged too.

### CODE REVIEW

File: tgm-example-plugin.php [hunk 1]
```diff
@@ -1,43 +1,68 @@
...
  *
- * @since 1.0.0
+ * @package     WordPress\Plugins\TGM Example Plugin
+ * @author      Thomas Griffin <thomas@thomasgriffinmedia.com>
+ * @link        https://github.com/thomasgriffin/TGM-Plugin-Activation
+ * @version     1.0.1
+ * @copyright   2011-2016 Thomas Griffin
+ * @license     http://creativecommons.org/licenses/GPL/3.0/ GNU General Public License, version 3 or higher
  *
+ * Plugin Name: TGM Example Plugin
```
Comment (GaryJones) [inline tgm-example-plugin.php:15]: The `@wordpress-plugin` tag is needed before this line. Non-tagged comments are not valid after PHPDoc tags.

**Comment by GaryJones** [comment]
Merged, but I'd like to see a README.md added to this repo to explain what the repo is.

**Comment by jrfnl** [comment]
In the absence of a `readme.md`, the `readme.txt` file which I added will display by default on the main page. Will that do ?

**Comment by GaryJones** [comment]
I just can't stand the look of .txt for readme's on GitHub :-)

**Comment by jrfnl** [comment]
_grin_ I can get that, still how much effort do we want to put into an example plugin which is only intended to demonstrate the capabilities of TGMPA ?

**Pull request merged**